openlibrary/components/MergeUI/AuthorRoleTable.vue

```diff
@@ -19,6 +19,9 @@
                                 {{role[field].key.slice("/authors/".length)}}
                             </a>
                         </div>
+                        <div v-else-if="field == 'name'">
+                            <b>{{role[field]}}</b>
```

Comment:
Not sure how this is done elsewhere in the codebase, but this _feels_ like something where a `class` property (on the div, if one isn’t already assigned?) and CSS rule would be more appropriate for formatting than using `<b>`.